docker/py310_wheel.docker

```diff
@@ -0,0 +1,79 @@
+FROM quay.io/pypa/manylinux2014_x86_64
+
+# set python version
+ENV PYMAJOR 3
+ENV PYMINOR 10
+ENV TARGET cp${PYMAJOR}${PYMINOR}-cp${PYMAJOR}${PYMINOR}${PYUNICODE}
+
+# boost version
+ENV BOOSTMAJOR 79
+ENV BOOSTMINOR 0
+ENV BOOST 1.${BOOSTMAJOR}.${BOOSTMINOR}
+ENV BOOST_ 1_${BOOSTMAJOR}_${BOOSTMINOR}
+
+# where do we epect casacore data build time and runtime
+ENV CASACORE_DATA /usr/share/casacore/data
+
+# how many threads to use for compiling
+ENV THREADS 4
+
+# install rpms
+RUN yum install -y flex cfitsio-devel blas-devel lapack-devel ncurses-devel readline-devel fftw-devel wcslib-devel gsl-devel
+
+# download other source code
+WORKDIR /tmp
+RUN curl http://www.iausofa.org/2015_0209_F/sofa_f-20150209_a.tar.gz --output /tmp/sofa.tgz
+RUN curl ftp://ftp.astron.nl/outgoing/Measures/WSRT_Measures.ztar --output /tmp/measures.tgz
+RUN curl https://ufpr.dl.sourceforge.net/project/boost/boost/${BOOST}/boost_${BOOST_}.tar.bz2 --output /tmp/boost.tar.bz2
+
+RUN mkdir /build
+WORKDIR /build
+
+RUN mkdir -p ${CASACORE_DATA}
+WORKDIR ${CASACORE_DATA}
+RUN tar zxvf /tmp/measures.tgz
+
+# install and configure sofa and measures
+WORKDIR /build
+RUN tar zxvf /tmp/sofa.tgz
+WORKDIR /build/sofa/20150209_a/f77/src
+RUN make -j${THREADS}
+
+
+# setup boost
+WORKDIR /build
+RUN tar jxvf /tmp/boost.tar.bz2
+WORKDIR /build/boost_${BOOST_}
+# patch for python 3.10 (multiple digits)
+#sed -i 's/(\[0-9\])/([0-9][0-9]?)/' tools/boost_install/BoostConfig.cmake
+RUN ./bootstrap.sh --prefix=/opt/boost \
+    --with-libraries=python \
+    --with-python=/opt/python/${TARGET}/bin/python \
+    --with-python-version=${PYMAJOR}.${PYMINOR} \
+    --with-python-root=/opt/python/${TARGET}
+RUN ./b2 -j${THREADS} \
+    cxxflags="-fPIC -I/opt/python/${TARGET}/include/python${PYMAJOR}.${PYMINOR}${PYUNICODE}/" \
+    link=static,shared install
+
+# casacore wants numpy
+RUN /opt/python/${TARGET}/bin/pip install numpy==1.22.4
+
+# set up casacore
+ADD . /code
+RUN useradd -ms /bin/bash casacore
+RUN chown casacore.casacore /code
+RUN mkdir /code/build
```

Comment:
The existing py3x_wheel.docker files have `USER casacore` above this line. Why did you remove it for python3.10?

Reply:
I removed it because sometimes it's annoying to not be root. But for consistency I put the line back, thanks.